src/pages/Details.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import NavBar from '../component/nav/navBar';
+import PlaneDetails from '../component/plane/planeDetails';
+import Layout from '../component/wrapper/Layout';
+
+const Details = () => (
+  <Layout preAnimation="fadeIn">
+    <div style={{ display: 'flex', flexDirection: 'row', width: '100vw' }}>
```

Comment:
- [ ] If you can use classes for styling here, it would be nice, since it's not a good practice to use the `style` attribute.